mongoengine/document.py

```diff
@@ -320,8 +321,6 @@ def save(self, force_insert=False, validate=True, clean=True,
 
         try:
             collection = self._get_collection()
-            if self._meta.get('auto_create_index', True):
```

Comment:
@wojcikstefan `self._get_collection()` does this too, so I removed the duplicate
https://github.com/MongoEngine/mongoengine/blob/de18e256cebd46e9412274dd73704c0b8fc18cb9/mongoengine/document.py#L198